README.md

```diff
@@ -1,64 +1,31 @@
 linter-foodcritic
 =========================
 
-This linter plugin for [Linter](https://github.com/AtomLinter/Linter) provides an interface to [foodcritic](https://github.com/AtomLinter/linter-foodcritic). It will be used with files that have the “Ruby” syntax.
+This linter plugin for [Linter](https://github.com/atom-community/linter)
```

Comment:
I think I see what you mean, I was planning on further cleaning up the actual text when (if...) further work was done on this like auto-installation of `linter`.